resource-managers/mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterSchedulerSuite.scala

```diff
@@ -200,6 +201,60 @@ class MesosClusterSchedulerSuite extends SparkFunSuite with LocalSparkContext wi
     })
   }
 
+  test("supports spark.mesos.driver.memoryOverhead with 384mb default") {
```

Comment:
```suggestion
  test("SPARK-22256: supports spark.mesos.driver.memoryOverhead with 384mb default") {
```

Reply:
Added this ticket number to both test cases.